### PR TITLE
CU-2p64ar7 - Update custom rpc wrapper representation

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**
## Issue
Fix the `assets.balanceOf` custom RPC by moving the expected JSON representation from hex string to string (Display).

## Checklist

- [x] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [x] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2p64ar7 Asset RPC balanceOf](https://app.clickup.com/t/2p64ar7)

File: frame/composable-support/src/rpc_helpers.rs [hunk 1]
```diff
@@ -1,79 +1,42 @@
...
+	Eq,
+	MaxEncodedLen,
+	Ord,
+	PartialEq,
+	PartialOrd,
+)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-#[derive(Debug, PartialEq, Eq, Encode, Decode)]
-pub struct SafeRpcWrapper<T: SafeRpcWrapperType>(
-	#[cfg_attr(feature = "std", serde(serialize_with = "serialize_to_hex"))]
```
Comment (vimukthi-git) [inline frame/composable-support/src/rpc_helpers.rs:9]: What was the reason to use hex representation before in the first place?
I guess I need to change [my PR](https://github.com/ComposableFi/composable/pull/900) based on this as well

File: frame/composable-support/src/rpc_helpers.rs [hunk 2]
```diff
@@ -1,79 +1,42 @@
-use codec::{Codec, Decode, Encode};
+use codec::{Decode, Encode, MaxEncodedLen};
 #[cfg(feature = "std")]
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
-
-/// https://github.com/interlay/interbtc/blob/a7c0e69ac041176a2531bafb1c4e35cbc2f7e192/crates/oracle/rpc/runtime-api/src/lib.rs#L10
```
Comment (vimukthi-git) [inline frame/composable-support/src/rpc_helpers.rs:5]: Original issue mentioned in this link https://github.com/paritytech/substrate/issues/4641 seems still relevant right? i.e the JSON serde has issues with `u128`

File: frame/composable-support/src/rpc_helpers.rs (hunk 1 above)
Comment (hussein-aitlahcen) [reply frame/composable-support/src/rpc_helpers.rs:9]: I'm not sure tbh.

File: frame/composable-support/src/rpc_helpers.rs (hunk 2 above)
Comment (hussein-aitlahcen) [reply frame/composable-support/src/rpc_helpers.rs:5]: Yeah so a number cannot be u128 in JSON thats the reason AFAIK

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by andor0 — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

File: frame/composable-support/src/rpc_helpers.rs (hunk 2 above)
Comment (vimukthi-git) [reply frame/composable-support/src/rpc_helpers.rs:5]: Normally big numbers are represented as string in JSON, it's strange that this is not getting fixed

**Pull request merged**